paddle/scripts/paddle_build.sh

```diff
@@ -405,17 +405,19 @@ EOF
 
 function gen_dockerfile() {
     # Set BASE_IMAGE according to env variables
+    CUDA_VERSION=$(nvcc --version | grep release | cut -d "," -f 2 | cut -d " " -f 3)
+    CUDNN_VERSION=$(cat /usr/include/cudnn.h | grep "#define CUDNN_MAJOR" | cut -d " " -f 3)
```

Comment:
this line does not work in paddlepaddle/paddle_manylinux_devel:cuda9.0_cudnn7
```bash
[root@e879a533d8f8 /]# cat /usr/include/cudnn.h | grep "#define CUDNN_MAJOR" | cut -d " " -f 3
cat: /usr/include/cudnn.h: No such file or directory
```

Reply:
I changed to use environment valuable.